Potentially a missing 'static somewhere

Hi @Baptistemontan 

We are in the process of migrating our code to leptos 0.7.7 (from 0.6) and we are hitting this error, which seems to come from i18n

```rust
error[E0310]: the parameter type `Chil` may not live long enough
 --> apis/src/lib.rs:9:1
  |
9 | leptos_i18n::load_locales!();
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | |
  | the parameter type `Chil` must be valid for the static lifetime...
  | ...so that the type `Chil` will meet its required lifetime bounds
  |
  = note: this error originates in the attribute macro `l_i18n_crate::reexports::leptos::component` which comes from the expansion of the macro `leptos_i18n::load_locales` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider adding an explicit lifetime bound
 --> /Users/leex/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/leptos_i18n-0.5.5/src/macros.rs:10:67
  |
10|         $crate::__private::macros_reexport::load_locales!{$($tt)*}: 'static
  |                                                                   +++++++++

error[E0310]: the associated type `<impl leptos::IntoView as leptos::prelude::Render>::State` may not live long enough
 --> apis/src/lib.rs:9:1
  |
9 | leptos_i18n::load_locales!();
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | |
  | the associated type `<impl leptos::IntoView as leptos::prelude::Render>::State` must be valid for the static lifetime...
  | ...so that the type `<impl leptos::IntoView as leptos::prelude::Render>::State` will meet its required lifetime bounds
  |
  = help: consider adding an explicit lifetime bound `<impl leptos::IntoView as leptos::prelude::Render>::State: 'static`...
  = note: this error originates in the attribute macro `l_i18n_crate::reexports::leptos::component` which comes from the expansion of the macro `leptos_i18n::load_locales` (in Nightly builds, run with -Z macro-backtrace for more info)
```

If you would like to reproduce it locally check out https://github.com/hiveboardgame/hive/pull/427
We had the same problem in places and `+ 'static` fixed it for our components.

Any help here would be appreciated! 😄

--- a/leptos_i18n_macro/src/load_locales/mod.rs
+++ b/leptos_i18n_macro/src/load_locales/mod.rs
@@ -185,7 +185,7 @@ fn load_locales_inner(
             /// Create and provide a i18n context for all children components, directly accessible with `use_i18n`.
             #[l_i18n_crate::reexports::leptos::component]
             #[allow(non_snake_case)]
-            pub fn I18nContextProvider<Chil: IntoView>(
+            pub fn I18nContextProvider<Chil: IntoView + 'static>(
                 /// If the "lang" attribute should be set on the root `<html>` element. (default to true)
                 #[prop(optional)]
                 set_lang_attr_on_html: Option<bool>,
@@ -221,7 +221,7 @@ fn load_locales_inner(
             /// Create and provide a subcontext for all children components, directly accessible with `use_i18n`.
             #[l_i18n_crate::reexports::leptos::component]
             #[allow(non_snake_case)]
-            pub fn I18nSubContextProvider<Chil: IntoView>(
+            pub fn I18nSubContextProvider<Chil: IntoView + 'static>(
                 children: TypedChildren<Chil>,
                 /// The initial locale for this subcontext.
                 /// Default to the locale set in the cookie if set and some,

